### ISSUE TITLE
What should we do with pixels with large negative dark current?

### ISSUE THREAD

**Issue by jeanconn** [sot/proseco#265]
For star selection and review how should a pixel with a reported value like the -8570 below be handled?   Is it a bad/broken pixel that should just be completely avoided for guide stars and fids?   Or should it just be set to 0 so it doesn't cause numeric issues?

```
ipdb> img
<ACAImage row0=-215 col0=339
array([[  142,   147,    12,    42,    14,    13,    21,   190],
       [   18,    34,    36,   524,    83,    12,    28,    26],
       [ -408, -8570,    26,    42,    29,   126,    26,    25],
       [  179,   183,    85,    68,   359,    21,    24,    17],
       [  187,     0,    38,   612,    30,    27,     8,     9],
       [   27,   -12,    17,    86,    35,   153,    19,   133],
       [   16,    57,    11,    16,    18,    27,    62,    27],
       [   27,   205,    27,    18,    69,    15,    23,     1]])>
```

**Comment by taldcroft** [comment]
Going OT, at some point in the not-too-distant future you should process a couple of SRDCs and see if this value persists.  Most likely not since it arises from a large value in im1.fits in 2019:002 and is not see in im2.fits.  My recollection is that the 5-sec read generally occurs before the 10 sec, so it is hard to image this is real.  Nevertheless...

It is fair to just clip at 0.0.  I'm almost inclined to put that upstream in mica dark_cal as a default value for clipping (with the option to set to None for no clipping).

But in any case clipping to 0 in proseco is just fine.  The real dark current cannot be negative.

**Comment by jeanconn** [comment]
"My recollection is that the 5-sec read generally occurs before the 10 sec, so it is hard to imagine this is real. Nevertheless..."  I just couldn't wrap my mind around the pixel having a large value in the 5 second readout and a small value in 10 second readout for each replica.

**Comment by jeanconn** [comment]
And I had the SRDC thought as well when we were wondering about the star selection, but we'll get there eventually.

**Comment by jeanconn** [comment]
Closed by mica 3.16 .
